packages/svelte-check/package.json

```diff
@@ -27,11 +27,11 @@
         "chokidar": "^3.4.1",
         "picocolors": "^1.0.0",
         "sade": "^1.7.4",
-        "svelte-preprocess": "^5.1.3",
-        "typescript": "^5.0.3"
+        "svelte-preprocess": "^5.1.3"
     },
     "peerDependencies": {
-        "svelte": "^3.55.0 || ^4.0.0-next.0 || ^4.0.0 || ^5.0.0-next.0"
+        "svelte": "^3.55.0 || ^4.0.0-next.0 || ^4.0.0 || ^5.0.0-next.0",
+        "typescript": ">=5.0.0"
```

Comment:
Should we support some versions of Typescript 4? It might be nice for users who haven't been able to upgrade yet if it's not too hard for us

Reply:
Thanks for reminding me to bump the Svelte peer dependency to v4 😄 And since v4 has an implicit dependency requirement on TS >=5 we should, too. Better to nudge users towards upgrading

Reply:
I think users will have plenty of reason to upgrade on their own since Svelte 5 is so much better. But it'd probably be appreciated if we let them do it in their own time instead of forcing them. It can make things easier to have wider support as you have more flexibility around what order to upgrade various libraries

Reply:
This isn't forcing people to jump to Svelte 5, only Svelte 4 - which you need to do before upgrading to 5 anyway.

Reply:
yeah, I know. I was just saying people would want to upgrade to Svelte 4 in order to get to 5 so we don't need to push them too hard